commands/deploy.go

```diff
@@ -132,10 +149,17 @@ func init() {
 		"template-file",
 		"t",
 		"",
-		"Path to the CloudFormation template to be deployed",
+		"Path to the CloudFormation template to be deployed.  Must not specify template-url.",
 	)
 	deployCmd.MarkFlagFilename("template-file")
 
+	deployCmd.PersistentFlags().StringVar(
+		&templateUrl,
+		"template-url",
+		"",
+		"S3 url to the CloudFormation template to be deployed.  Must not specify template-file.",
```

Comment:
Could we perhaps reword this to the following? 
```suggestion
		"S3 URI of the CloudFormation template to be deployed (This argument conflicts with \"template-file\")",
```